catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/ExtendedGeotoolsFunctionFactory.java

```diff
@@ -24,20 +24,29 @@
 import org.opengis.filter.expression.Function;
 import org.opengis.filter.expression.Literal;
 
+import ddf.catalog.impl.filter.DivisibleByFunction;
+import ddf.catalog.impl.filter.ProximityFunction;
+
 /**
  * The ExtendedGeotoolsFunctionFactory is used to provide the GeoTools CommonFactoryFinder a list of custom
  * functions.  This allows for DDF to extend the OGC Filter 1.1.0 schema and pass along the extension
  * changes to Geotools via a service loader .
  */
+//TODO (RWY) - can we get rid of this and use the one in geotools?
 public class ExtendedGeotoolsFunctionFactory implements FunctionFactory {
```

Comment:
I don't think their is any reason to have this class in addition to the GeotoolsFunctionFactory but I would have to move PropertyIsFuzzy down into geotools in order to keep fuzzy search in csw backwords compatible